src/pages/lesson-details/LessonDetails.tsx

```diff
@@ -80,6 +90,9 @@ const LessonDetails = () => {
 
   return (
     <PageWrapper>
+      <AppButton onClick={handleEditLesson} sx={styles.button}>
+        {t('common.edit')} <EditIcon sx={styles.editIcon} />
+      </AppButton>
```

Comment:
<AppButton onClick={handleEditLesson} sx={styles.button}>
        <Typography>{t('common.edit')}</Typography>
        <EditIcon sx={styles.editIcon} />
      </AppButton>

Set:     display: 'flex', alighnItems: 'center', columnGap to AppButton

Reply:
done